src/api/config/clock.rb

```diff
@@ -48,6 +48,10 @@ module Clockwork
     BsRequest.delayed_auto_accept
   end
 
+  every(1.hour, 'refresh remote distros') do
+    FetchRemoteDistributionsJob.perform_later
```

Comment:
why this is necessary after this PR and it wasn't needed before it?

Reply:
Before we stored the `distribution.xml` from the  interconnect projects in memcache. See

https://github.com/openSUSE/open-build-service/pull/11223/files#diff-66b82a7133958122ff9b8c0436aba2f4a3eef044f869b90c61209221afc288f4L54-L57

Reply:
then converted XML to XMLHash in the `DistributionController`. Used that XMLHash in the XML views.

Which also means we where processing the Hash in `RepositoryController` when displaying them in the UI yadda yadda yadda. Hash overflow....

Reply:
> Before we stored the `distribution.xml` from the interconnect projects in memcache

Isn't this approach better? Why should we keep updating it every hour regardless of traffic or not?

Reply:
> Isn't this approach better? Why should we keep updating it every hour regardless of traffic or not?

Better how?